src/App.tsx

```diff
@@ -17,18 +69,20 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter filter={handleSelect} setSearchQuery={setSearchQuery} searchQuery={searchQuery} clearSearch={clearSearch} />
```

Comment:
I see a lot of linter problems in your code, please fix it
```suggestion
      <TodoFilter
        filter={handleSelect}
        setSearchQuery={setSearchQuery}
        searchQuery={searchQuery}
        clearSearch={clearSearch}
      />
```